### PR TITLE
Sort attributes in alphabetical order

### PR DESCRIPTION
**Pull request by themr0c**
Sort attributes in alphabetical order

### CODE REVIEW

**Review by yhontyk — APPROVED** [review]

File: src/main/_config.yml [hunk 1]
```diff
@@ -52,66 +52,58 @@ asciidoc_attributes: &asciidoc_attributes
...
-  repo-path: ''
-
-# stack names
   nodejs-stack: nodejs
-
   oc4-ver: 4.4
-  ocp: OpenShift{nbsp}Container{nbsp}Platform
   ocp3-ver: 3.11
   ocp4-ver: 4.5
+  ocp: OpenShift{nbsp}Container{nbsp}Platform
```
Comment (benoitf) [inline src/main/_config.yml:78]: not sorted ?

File: src/main/_config.yml [hunk 2]
```diff
@@ -52,66 +52,58 @@ asciidoc_attributes: &asciidoc_attributes
...
   orch-cli: kubectl
   orch-name: Kubernetes
   orch-namespace: namespace
   osd: OpenShift{nbsp}Dedicated
+  postgresql-image-url: quay.io/eclipse/che-postgres
+  prod2: Eclipse{nbsp}Che
   prod-checluster: eclipse-che
   prod-cli: chectl
   prod-deployment: che
+  prod: Eclipse{nbsp}Che
```
Comment (benoitf) [inline src/main/_config.yml:88]: looks like this one is not sorted

Comment (themr0c) [reply src/main/_config.yml:88]: That's the result of sorting with the `sort` command ...

File: src/main/_config.yml (hunk 1 above)
Comment (themr0c) [reply src/main/_config.yml:78]: That's the result of sorting with the `sort` command ...

File: src/main/_config.yml (hunk 2 above)
Comment (benoitf) [reply src/main/_config.yml:88]: I don't know which `sort` command


```
$ cat /tmp/list| sort                                                                                                                                                          
  oc4-ver: 4.4
  ocp: OpenShift{nbsp}Container{nbsp}Platform
  ocp3-ver: 3.11	  ocp3-ver: 3.11
  ocp4-ver: 4.5	  ocp4-ver: 4.5
  orch-cli: kubectl	  orch-cli: kubectl
  orch-name: Kubernetes	  orch-name: Kubernetes
  orch-namespace: namespace	  orch-namespace: namespace
  osd: OpenShift{nbsp}Dedicated	  osd: OpenShift{nbsp}Dedicated
  postgresql-image-url: quay.io/eclipse/che-postgres
  prod: Eclipse{nbsp}Che
  prod2: Eclipse{nbsp}Che
  prod-checluster: eclipse-che	  prod-checluster: eclipse-che
  prod-cli: chectl	  prod-cli: chectl
  prod-deployment: che	  prod-deployment: che
  prod-host: che-host	  prod-host: che-host
```

or with `-d`
```
$ cat /tmp/list| sort                                                                                                                                         
  oc4-ver: 4.4
  ocp: OpenShift{nbsp}Container{nbsp}Platform
  ocp3-ver: 3.11	  ocp3-ver: 3.11
  ocp4-ver: 4.5	  ocp4-ver: 4.5
  orch-cli: kubectl	  orch-cli: kubectl
  orch-name: Kubernetes	  orch-name: Kubernetes
  orch-namespace: namespace	  orch-namespace: namespace
  osd: OpenShift{nbsp}Dedicated	  osd: OpenShift{nbsp}Dedicated
  postgresql-image-url: quay.io/eclipse/che-postgres
  prod: Eclipse{nbsp}Che
  prod2: Eclipse{nbsp}Che
  prod-checluster: eclipse-che
  prod-cli: chectl	  prod-cli: chectl
  prod-deployment: che	  prod-deployment: che
  prod-host: che-host	  prod-host: che-host
```

**Pull request merged**